integration-libs/opf/occ/utils/opf-occ-http-error-handlers.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HttpErrorModel } from '@spartacus/core';
+
+export function isHttp500Error(err: HttpErrorModel): boolean {
+  return err?.status && err.status >= 500 ? true : false;
```

Comment:
will be enough `return !!err?.status && err.status >= 500;` or `return Boolean(err?.status) && err.status >= 500;`, without ternary operator.

Reply:
Thanks! change done.